src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.yml

```diff
@@ -27,6 +27,55 @@ injections:
         end: '(\})'
         endCaptures:
           1: { name: 'punctuation.curlybrace.close.cs' }
+  # 'L:meta.attribute':
+  #   patterns:
+  #     - name: 'meta.attribute.razor.explicit'
+  #       begin: '(")(@)\('
+  #       beginCaptures:
+  #         0: { name: 'keyword.control.cshtml' }
+  #         1: { patterns: [ include: '#transition' ] }
+  #       # begin: '"(@)\('
+  #       # beginCaptures:
+  #       #   0: { name: 'keyword.control.cshtml' }
+  #       #   1: { patterns: [ include: '#transition' ] }
+  #       patterns:
+  #         - include: 'source.cs#expression'
+  #       end: '\)'
+  #       endCaptures:
+  #         0: { name: 'keyword.control.cshtml' }
+  #     - name: 'meta.attribute.razor.implicit'
+  #       begin: '("[^@]*)(@)'
+  #       beginCaptures:
+  #         0: { name: 'keyword.control.cshtml' }
+  #         1: { patterns: [ include: '#transition' ] }
+  #       patterns:
+  #         - include: '#await-prefix'
+  #         - include: '#implicit-expression-body'
+  #       end: '(?=[\s<>\{\}\)\]])(?=([^"]*")'
+
+
+  'L:meta.attribute string.quoted.double.html':
+    patterns:
+      - name: 'meta.attribute.razor.explicit'
+        begin: '@\('
+        beginCaptures:
+          0: { name: 'keyword.control.cshtml' }
+          1: { patterns: [ include: '#transition' ] }
+        patterns:
+          - include: 'source.cs#expression'
+        end: '\)'
+        endCaptures:
+          0: { name: 'keyword.control.cshtml' }
+      - name: 'meta.attribute.razor.implicit'
+        begin: '@'
+        beginCaptures:
+          0: { name: 'keyword.control.cshtml' }
+          1: { patterns: [ include: '#transition' ] }
```

Comment:
ditto

Reply:
Updated, thanks (removed `1: { patterns: [ include: '#transition' ] }` as it proved to be redundant).